dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/export/event/EventOperationParamsMapper.java

```diff
@@ -286,7 +286,7 @@ private String getAccessibleScopeValidation(User user, Program program) {
       return "User is required for orgUnitMode: " + OrganisationUnitSelectionMode.ACCESSIBLE;
     }
 
-    if (program == null || program.isClosed() || program.isProtected()) {
+    if (program != null && (program.isClosed() || program.isProtected())) {
```

Comment:
was that wrong?

Reply:
in case the program is null, we need to use the search scope by default